torch/fx/experimental/symbolic_shapes.py

```diff
@@ -195,6 +196,24 @@ def guard_scalar(a):
 
 # inclusive both ways
 def constrain_range(a, *, min: Optional[int], max: Optional[int] = None):
+    """
+    Constrain range takes a symbol, and records a valid value range for it in the graph.
```

Comment:
It doesn't take a Symbol; it takes a SymInt or int.

Reply:
thanks